src/index.html

```diff
@@ -13,6 +13,88 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+      <li class="stars__star"></li>
+
+    </ul>
```

Comment:
```suggestion
    <ul class="stars stars--0">
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
    </ul>
```

Here spaces between siblings are not needed, because it's just 1 line
The same below